Assets/World/AStarMap.gd

```diff
@@ -0,0 +1,80 @@
+extends Spatial
+
+var all_points = {}
+var as_node: AStar = null
+
+onready var grid_map = $GridMap
+
+var ray_plane := MeshInstance.new() as MeshInstance
+
+func _ready() -> void:
+	ray_plane.mesh = PlaneMesh.new()
+	ray_plane.visible = false
+
+	ray_plane.scale = Vector3(128, 128, 128) # TODO: Adapt size dynamically to GridMap's outer bounds
+
+	#ray_plane.create_convex_collision() # crashes for GLES2 in the export build
+#	var static_body = StaticBody.new()
+#	var collision_shape = CollisionShape.new()
+#	collision_shape.shape = PlaneShape.new()
```

Comment:
Inconsistent tabbing in these comments. I think it's better to place `#` after the tabs.

Reply:
The three other lines are a alternative approach to the buggy one-liner above. I made it consistent and put a blank line instead to make it clear they don't belong together.